docs/site-example/site.conf

```diff
@@ -1,4 +1,4 @@
--- This is an example site configuration for Gluon v2022.1.1
```

Comment:
If this site.conf gets bumped, the one in https://github.com/freifunk-gluon/gluon/blob/0a2eac6e5d8c1940749b0dc1b53d0c254e6295d8/contrib/ci/minimal-site/site.conf#L1
should be as well, right?
If so, we should add it to the checklist.